src/arch/riscv/sbi.c

```diff
@@ -476,5 +476,7 @@ void sbi_init()
         }
     }
 
-    interrupts_reserve(TIMR_INT_ID, sbi_timer_irq_handler);
+    if(!interrupts_reserve(TIMR_INT_ID, sbi_timer_irq_handler)) {
+        ERROR("Failed to assign SBI TIMR_INT_ID interrupt");
```

Comment:
```suggestion
        ERROR("Failed to reserve SBI TIMR_INT_ID interrupt");
```

Reply:
Thank you, fixed in https://github.com/bao-project/bao-hypervisor/pull/97/commits/878ad2b2ed85b8ed383a63e32650ee9363c624d8.